### PR TITLE
Telemetry licensing

### PR DESCRIPTION
**Pull request by RaunakShah**
Adds the correct way to update the telemetry configuration in MKE4.

### CODE REVIEW

File: content/docs/operations/telemetry.md [hunk 1]
```diff
@@ -12,12 +12,14 @@ services.
 
 ## Enable telemetry through the MKE CLI
 
-Go to the `mkeconfig` object in the `mke` namespace and set `tracking.enabled` to `true`.
+To enable telemetry for MKE 4, set the `tracking.enabled` section of the MKE configuration file.
```
Comment (KoryKessel-Mirantis) [inline content/docs/operations/telemetry.md:15]:
```suggestion
You enable telemetry for MKE 4 through the `tracking.enabled` section of the MKE configuration file:
```

**Comment by KoryKessel-Mirantis** [comment]
Hey @RaunakShah, can you please take a look at https://github.com/Mirantis/mke-docs/issues/132 and perhaps drop a rough note into this PR that addresses @brianbancroft's ask (which, of course, I will be glad to help sharpen up)?

**Comment by RaunakShah** [comment]
Didn't see that https://github.com/Mirantis/mke-docs/pull/125 addresses the same thing as this PR. So I've updated it to only include the licensing part.

File: content/docs/operations/telemetry.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ use MKE. It also provides product usage statistics, which is key feedback that
 helps product teams in their efforts to enhance Mirantis products and
 services.
 
+{{< callout type="note" >}}
+    In case the license associated with the MKE cluster is invalid or expired, telemetry will be enabled by default and there will be no option to disable it.
```
Comment (KoryKessel-Mirantis) [inline content/docs/operations/telemetry.md:14]:
```suggestion
    Telemetry is automatically enabled for MKE 4 clusters that are running without a license, with a license that has expired, or with an invalid license. In all of such scenarios, you will only be able to disable telemetry once a valid license has been applied to the cluster.
```

**Review by KoryKessel-Mirantis — CHANGES_REQUESTED** [review]

**Review by KoryKessel-Mirantis — APPROVED** [review]

**Pull request merged**